extension-jaxrs/src/test/resources/io/smallrye/openapi/runtime/scanner/params.parameter-ref-property.json

```diff
@@ -54,9 +54,6 @@
               "type": "string"
             }
           },
-          {
-            "$ref": "#/components/parameters/pathParam2"
-          },
```

Comment:
This should have been hidden all along (line 32)
https://github.com/smallrye/smallrye-open-api/blob/9804d391d20b9a8c9315102d05b9226d452ee4da/extension-jaxrs/src/test/java/test/io/smallrye/openapi/runtime/scanner/jakarta/ParameterRefTestResource.java#L22-L34